src/requests/team.ts

```diff
@@ -18,21 +19,23 @@ interface RemoveMember {
 
 type PlanXEnv = "pizza" | "staging" | "production";
 
+interface CreateTeam {
```

Comment:
We could also make `theme` an optional argument here in future if we asked this in the future when teams can be created within PlanX